internal/pkg/state/local/create.go

```diff
@@ -19,6 +19,16 @@ func (m *Manager) createObject(key model.Key, name string) (model.Object, error)
 		if err != nil {
 			return nil, err
 		}
+
+		// wrap config content to parameters
+		// { "parameters":{...}
+		if content.Len() != 0 {
+			finalContent := orderedmap.New()
+			finalContent.Set("parameters", content)
+
+			content = finalContent
+		}
```

Comment:
Why it was not done in `generateContent`? Looks like it operates with content and fills the `parameters` for both cases.

Reply:
[f8f88a4](https://github.com/keboola/keboola-as-code/pull/1940/commits/f8f88a43c471bfe689b0c051b9d70f82ad9556c1)